client/src/utils/OrDivider/OrDivider.tsx

```
@@ -0,0 +1,11 @@
+const OrDivider = () => {
+  return (
+    <div className="relative flex py-5 items-center">
+      <div className="flex-grow border-t border-gray-300"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)